deploy/charts/klicker-uzh-v2/templates/deployment-hatchet-workers.yaml

```diff
@@ -21,7 +21,6 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/cm-hatchet-workers.yaml") . | sha256sum }}
-        checksum/secret: {{ include (print $.Template.BasePath "/secret-hatchet-workers.yaml") . | sha256sum }}
```

Comment:
how do we make sure that deployments are recreated if only secrets change? does the ESO do this cleanly?